platform/admin/ui/src/main/webapp/main.js

```diff
@@ -159,33 +159,35 @@
             }));
         });
 
-        var alerts = new AlertsModel.InsecureAlerts();
-
-        alerts.fetch({
-            success: function () {
-                app.addInitializer(function () {
-                    Application.App.alertsRegion.show(new AlertsView.View({
-                        model: alerts
-                    }));
-                });
-            }
-        });
+        // setup alert banners
+        app.addInitializer(function () {
+            var alerts = new Backbone.Collection([]);
 
-        //refresh the session if keypress  or mouse click events occur
-        SessionRefresherUtil(60000);
+            var backendAlerts = new AlertsModel.BackendAlerts();
+            backendAlerts.fetch();
```

Comment:
❓ Is there any chance this backendAlerts collection would be substantial in size or have any kind of delay while being fetched?